OrbitGl/CaptureWindow.cpp

```diff
@@ -526,13 +526,16 @@ void CaptureWindow::Draw() {
     // We use different coordinate systems for ScreenSpace items (margin, scrollbar, ...)
     // and for the text than the rest of items inside CaptureWindow. So, we have to switch
     // between these 2 systems while the layer is changing (with these "Prepare.." functions).
-    if (layer < GlCanvas::kScreenSpaceCutPoint) {
+
+    // Lines from TimeBar are also drawn in WorldSpace coordinate system
+    if (layer < GlCanvas::kScreenSpaceCutPoint || layer == GlCanvas::kZValueTimeBar) {
```

Comment:
Hmmm, this seems a bit weird. I think the problem is that the time bar's background is rendered in screen space while the time lines are rendered in world space. They should be rendered in the same coordinate system, then their z-values will be on the same side of the "kScreenSpaceCutPoint" delimiter.

Reply:
I agree, we should try to not introduce special cases like this after we've just spent time on simplifying the z-layering. This will just make it harder to catch bugs in the future.

If a good fix takes more time, I'd rather not fix it for the next release and fix it properly, because once this hack is in, other priorities may come along and this stays in forever until it causes issues.

Reply:
Yes, the problem is exactly that.

Ok. I didn't have time today to make it, but I will try to draw both of them in ScreenSpace coordinate system, which for me is what has more sense. 

Thanks!